packages/reference/src/common/useContentTypePermissions.ts

```diff
@@ -56,12 +60,17 @@ export function useContentTypePermissions(
 
     async function checkContentTypeAccess() {
       const creatable = await getContentTypes('create');
-      setCreatableContentTypes(creatable);
+      // Important as `filter` creates a new array and otherwise always a "new value" would be written to the state
+      if (!isEqual(creatable, creatableContentTypes)) {
+        setCreatableContentTypes(creatable);
+      }
     }
 
-    void checkContentTypeAccess();
+    if (availableContentTypes.length > 0) {
+      void checkContentTypeAccess();
+    }
     // eslint-disable-next-line react-hooks/exhaustive-deps -- TODO: Evaluate the dependencies
-  }, [availableContentTypes]);
+  }, [availableContentTypes, creatableContentTypes]);
```

Comment:
I think there is a Typo and the var should be called createableContentTypes ? Or was that there before

Reply:
it's everywhere in this file without `e` after creat 🙈

Reply:
let me fix the typos :D

Reply:
we have the same working everywhere :/ so maybe not working fixing it now

Reply:
💔 i see, then we will live with it!